### PR TITLE
Fix field usage

### PR DESCRIPTION
**Pull request by elnur**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@elnur Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-stream&pullRequestId=713)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-stream&pullRequestId=713) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@elnur Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-stream&pullRequestId=713)!

**Comment by ilayaperumalg** [comment]
@elnur Thanks for the PR.

File: spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc [hunk 1]
```diff
@@ -382,7 +382,7 @@ public class SendingBean {
     }
 
     public void sayHello(String name) {
-         customOutput.send(MessageBuilder.withPayload(body).build());
+         output.send(MessageBuilder.withPayload(body).build());
```
Comment (ilayaperumalg) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: Even the variable `body` needs to use `name` instead. will fix while merging.

**Comment by ilayaperumalg** [comment]
LGTM, merged as 9b79a7c32a1483c6fd9266c9d3f142397ba7f4a3 along with 67a3e3c9151da81b5849352c257e46a6682305d8

**Pull request merged**